var/spack/repos/jcsda-emc-bundles/packages/base-env/package.py

```diff
@@ -21,6 +21,7 @@ class BaseEnv(BundlePackage):
     # Basic utilities
     if sys.platform == "darwin":
         depends_on("libbacktrace", type="run")
+    depends_on("bison@3.4:", type="run")
```

Comment:
Is bison getting used directly by, say, JCSDA developers, or is it just a dependency for packages that are directly used? If it's the latter, I might be inclined to put a default version in configs/common/package.py rather that in the metapackages.

Reply:
@srherbener Please correct me if I am wrong, I think bison is used in IODA.

Reply:
I am correcting myself after looking at the IODA repo and talking with @srherbener. We don't use bison explicitly, it's an indirect dependency. The trouble with the package versions in the common `packages.yaml` is that they are not hard requirements, just "recommendations". And we already have that set: https://github.com/JCSDA/spack-stack/blob/ff8fe8bad28e95e053fffbec74c764b6b2fd968f/configs/common/packages.yaml#L16